cdap-ui/app/cdap/components/PipelineSummary/RunsHistoryGraph/index.js

```diff
@@ -140,6 +144,19 @@ export default class RunsHistoryGraph extends Component {
       />
     );
   }
+  navigateToPipeline(runid) {
+    let anchor = document.getElementById('placeholder-anchor-runs-history');
+    let {namespaceId: namespace, appId: pipelineId} = this.props.runContext;
+    let url = window.getHydratorUrl({
+      stateName: 'hydrator.detail',
+      stateParams: {
+        namespace,
+        pipelineId
+      }
+    });
+    anchor.href = `${url}?runid=${runid}`;
+    anchor.click();
```

Comment:
Hmm so is the user actually clicking on the anchor at all? Why not use `window.location.href`?